### PR TITLE
Di 20298 editors editor general tab

### PR DESCRIPTION
**Pull request by michaelatias**

### CODE REVIEW

File: client-side/src/app/views-editor/views-editor.component.ts [hunk 1]
```diff
@@ -23,129 +24,39 @@ export class ViewsEditorComponent implements OnInit {
...
-  initDataSource(){
-    this.dataSource = {
-      Name: this.view.Name,
-      Description: this.view.Description,
-      Resource: this.view.Resource.Name
-    }
-  }
   onBackToList(){
-    this.router.navigate([".."], { relativeTo: this.route})
+    this.location.back()
```
Comment (roiaharon-pepperi) [inline client-side/src/app/views-editor/views-editor.component.ts:57]: why have you changed it to work with location?

**Review by roiaharon-pepperi — CHANGES_REQUESTED** [review]

File: client-side/src/app/views-editor/views-editor.component.ts (hunk 1 above)
Comment (michaelatias) [reply client-side/src/app/views-editor/views-editor.component.ts:57]: no idea why I changed it, cant remember why. I guess by mistake

File: client-side/src/app/editors-form/editors-form.component.html [hunk 2]
```diff
@@ -1 +1,56 @@
-<p>editors-form works!</p>
+<div class="header">
```
[the remark this answers is not in the document]
Comment (michaelatias) [reply client-side/src/app/editors-form/editors-form.component.html:1]: no idea how pep top bar will help me with buttons, I looked for example but couldn't find out.

File: client-side/src/app/editors/view-editor.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { View, Editor } from "../../../../shared/entities";
+import { AbstractEditor } from "./editor";
+
+export  class ViewEditor extends AbstractEditor{
+  content: View;
+  async initView(key: string){
+      if(key === "new"){
```
[the remark this answers is not in the document]
Comment (michaelatias) [reply client-side/src/app/editors/view-editor.ts:7]: deprecated

File: client-side/src/app/editors-form/editors-form.component.scss [hunk 4]
```diff
@@ -1,8 +1,3 @@
-.header{
-    display: flex;
-    justify-content: space-between;
-    margin-top: 1%;
-}
 
 .buttons-holder{
     float: right;
```
Comment (roiaharon-pepperi) [inline client-side/src/app/editors-form/editors-form.component.scss:3]: do you still need this css after you replaced the HTML?

File: client-side/src/assets/i18n/en.json [hunk 5]
```diff
@@ -11,5 +11,8 @@
     "Add": "Add",
     "Update": "Update",
     "Back_To_List": "Back to list",
+    "Design": "Design",
+    "Same-page": "replace the current page",
```
Comment (roiaharon-pepperi) [inline client-side/src/assets/i18n/en.json:15]: change 'Same-page' to 'Same_Page'

**Review by roiaharon-pepperi — APPROVED** [review]

**Pull request merged**